.github/workflows/issues.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2023 Stacklok, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This workflow labels issues that are opened or reopened with the needs-triage label.
+name: Label issues new issues with needs-triage
```

Comment:
I was imagining we would manually add the label when we think an issue / pr doesn't have enough information.

Reply:
Ah, I see. I'll remove the action then 👍 Yeah, I was thinking of having something automated that expects someone to go through the issue/PR and see if it's sensible or not. With these changes it doesn't make sense anymore 👍